File: sujet_machine/exemples/a_table/skeleton/a_table.php
<?php
function a_table0($n, $m, &$salle) {
    //  Inserez votre code ici 
}
list($n, $m) = array_map("intval", explode(" ", fgets(STDIN)));
$cw = array();
for ($cx = 0; $cx < $n; $cx++)
    $cw[$cx] = fgets(STDIN);
$salle = $cw;
a_table0($n, $m, $salle);

File: sujet_machine/exemples/a_table/skeleton/a_table.pl
#!/usr/bin/perl

sub a_table0{
  my($n, $m, $salle) = @_;
  # Inserez votre code ici 
  
}
my ($n, $m) = @{[ map { int($_) } split(/\s+/, <STDIN>) ]};
my $cy = [];
foreach my $cz (0 .. $n - 1)
{
    $cy->[$cz] = [split(//, <STDIN>)];
}
my $salle = $cy;
a_table0($n, $m, $salle);


